Hardware/Hardware/CardReader/CardBrand.swift

```diff
@@ -1,7 +1,7 @@
 import Codegen
 
 /// The various card brands for a card.
-public enum CardBrand: String, CaseIterable, Codable, GeneratedFakeable {
+@frozen public enum CardBrand: String, CaseIterable, Codable, GeneratedFakeable {
```

Comment:
I'm curious about the usage of `@frozen` here (never used it tbh 😅). Are we just telling the compiler that we're not adding more cases anytime soon?

Reply:
Yep, that is one of the things https://docs.swift.org/swift-book/documentation/the-swift-programming-language/attributes/#frozen